msg/DisplayRobotState.msg

```diff
@@ -4,5 +4,6 @@ RobotState state
 # Optionally, various links can be highlighted
 ObjectColor[] highlight_links
 
-# Optionally, hide the robot state (in rviz) if true
-bool hide
+# Optionally, do not display this state in visualizations
```

Comment:
I don't like the wording "_this state_" here, as - if hide is true - there is no need to provide a RobotState at all.

Reply:
Also _optional_ is kind of misleading here. An array can be empty (and thus optional), but a simple scalar variable cannot be optional. However, the default value of zero (`false`) is very useful here :wink: